lib/rate-limiter.js

```diff
@@ -60,11 +62,15 @@ class RateLimiter {
    *
    * @returns {Response}
    */
-  async handle (request, h) {
+  async handle(request, h) {
     if (this.isDisabledFor(request)) {
       return h.continue
     }
 
+    if (this.skip(request, h)) {
```

Comment:
Skipping rate limiting for this request is the same as disabling it. You can refactor the isDisabled handling and put the functionality for skip in there